tsl/src/bgw_policy/reorder_api.c

```diff
@@ -42,6 +41,35 @@
 #define DEFAULT_RETRY_PERIOD                                                                       \
 	DatumGetIntervalP(DirectFunctionCall3(interval_in, CStringGetDatum("5 min"), InvalidOid, -1))
 
+#define CONFIG_KEY_HYPERTABLE_ID "hypertable_id"
+#define CONFIG_KEY_INDEX_NAME "index_name"
+
+#define POLICY_REORDER_PROC_NAME "policy_reorder"
+
+int32
+policy_reorder_get_hypertable_id(BgwJob *job)
```

Comment:
This whole function seems unnecessary if we get the hypertable_id directly from the jobs table.